docs/products/opensearch/concepts/access_control.rst

```diff
@@ -43,43 +61,65 @@ The same set would deny the service user to
+::::::::
 
 * ``_*/admin`` would grant unlimited access to all top-level APIs
 * ``_msearch/admin`` grants unlimited access to the ``_msearch`` API only
 
+The ACL **only controls access to the API** and not what it can be used for. Giving access to the top-level API will in effect circumvent index specific rules, for example ``_msearch/admin`` access
+allows searching any index via the API as the indices to search are defined in the request body itself.
+
 .. note::
-   You might encounter ``HTTP 500`` internal server errors when you try to view dashboards as a service user that has read-only access to certain indexes, as these dashboards call the ``_msearch`` API. In such cases, add a new ACL rule that grants **Admin** access to ``_msearch`` for that service user.
+   When top-level API access is granted via explicit ACL the request content is not examined.
+
+Enabling extended ACLs
+++++++++++++++++++++++
 
-Only rules where the pattern starts with ``_`` are considered for top-level API access. Normal rules do not grant access to these APIs. For example, ``*search/admin`` only grants access to indexes that match the pattern, not to ``_msearch``.
+Instead of creating a rule that allows access to the top-level ``_mget``, ``_msearch`` and ``_bulk`` APIs, you can switch on the ``ExtendedAcl``.
+This will automatically enable these APIs for the user and each API request is checked to make sure operations only target indices
+that the user has appropriate permissions for (as defined by the normal index ACLs).
 
-You can switch on the ``ExtendedAcl`` option for the service to enforce index rules in a limited fashion for requests that use the ``_mget`` and ``_msearch`` APIs (and only those). When this option is in use, service users can access these APIs as long as all operations only target indexes that they have appropriate permissions for. 
+As the service must inspect the content of the request, this *can cause* performance and latency issues. The requests are also **limited to a maximum of 15000 bytes** in size.
+If the request is **too large** or if **any** of the operations or indices are not allowed by the ACLs, the *entire request* is rejected.
 
 .. note::
-   To enforce the rules with ``ExtendedACL``, the service must inspect the content of the request, which can cause performance and latency issues. All requests are also limited to a maximum of 16 KiB in size. If the request is too large or if any of the operations or indexes are not allowed, the entire request is rejected.
+   ACLs permitting access to top-level API will always take precedence over ``ExtendedACL``, you can for example allow access to ``_bulk`` for a trusted service account to
+   do mass updates.
 
 
 Access control and aliases
 --------------------------
 
-Aliases are not expanded. If you use aliases, the ACL must include a rule that matches the alias. 
+Aliases are **not expanded**. If you use aliases, the ACL must include a rule where the pattern matches the alias.
```

Comment:
I'm a bit puzzled to find this info here, should this be a warning when you talk about pattern/permissions?

Reply:
I moved this part higher as subsection to patterns and permissions section